cdap-ui/app/cdap/components/PaidBanner/index.tsx

```diff
@@ -0,0 +1,35 @@
+/*
```

Comment:
Since you're adding a new component, use CSS-in-JS to style the component instead of styling it from `PaidBanner.scss`. You can learn more [here](https://material-ui.com/styles/basics/) or search for `withStyles` in our code and look at any component that uses it.

Reply:
I was using the same structure that ExperimentalBanner uses. Should I also update that to use CSS-in-JS?

Reply:
Updated to CSS-in-JS